nodeEditor/src/diagram/graphFrame.ts

```diff
@@ -191,12 +194,9 @@ export class GraphFrame {
         if(!this.isCollapsed) {
             return;
         }
-        this.ports.forEach((framePort:FrameNodePort) => {
-            framePort.dispose()
-        });
-
-        this._createFramePorts();
-        this.ports.forEach((framePort: FrameNodePort) => framePort.node._refreshLinks());
+        this.isCollapsed = !this.isCollapsed
+        this.isCollapsed = !this.isCollapsed

```

Comment:
Suspicious

Reply:
Agreed! I was running into issues with some node links not being drawn correctly after other unrelated ones were deleted and this "hack" fixed them.

I wanted to come up with a better way before officially submitting this pr.